app/lib/config.js

```diff
@@ -1,4 +1,7 @@
 const config = {
+  app: {
```

Comment:
Given there is only a single instance of the app's name being used I don't see the rationale for adding it to the config.